pypeit/utils.py

```diff
@@ -185,7 +421,7 @@ def string_table(tbl, delimeter='print', has_header=True):
     return '\n'.join(row_string)+'\n'
 
 
-def spec_atleast_2d(wave, flux, ivar, gpm, copy=False):
+def spec_atleast_2d(wave, flux, ivar, gpm, log10_blaze_function=None, copy=False):
```

Comment:
Yeah I see the point. But this would be true for probably half the methods here. This method is used by two core  methods and one base class, so right now I'm inlaced to leave it. But I could move it to flux_calib.py for example.